Client/play/src/PlayerMap.jsx

```diff
@@ -18,6 +18,7 @@ module.exports = React.createClass({
   }, 
   componentDidMount: function() {
     gMap.startGMap({lng:-33.73, lat:149.02});
+    gMap.disableAddPins = true;
```

Comment:
correct, this will completely disable adding pins this is important for the player view. 
You will still be able to add pins manually or through logic it will just disable adding pins when/if the player presses on the screen as a creator would.